app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/FahrplanFragment.java

```diff
@@ -610,7 +610,7 @@ public void onClick(View v) {
 
     public void buildNavigationMenu() {
         Moment currentDate = Moment.now().startOfDay();
-        MyApp.LogDebug(LOG_TAG, "Today is " + currentDate.toUTCDateTime().toLocalDate());
+        MyApp.LogDebug(LOG_TAG, "Today is " + currentDate.toUtcDateTime().toLocalDate());
```

Comment:
FYI: This has been [discussed earlier](https://github.com/EventFahrplan/EventFahrplan/pull/245#discussion_r376616230) and postponed. I agree it is a good step to clean up the code base and to comply with the Java language conventions. :+1: Thank you.